vendor/engines/bulk_email/app/models/bulk_email/delivery_form.rb

```diff
@@ -43,9 +43,13 @@ def body(recipient_name = nil)
       content_generator.wrap_text(custom_message, recipient_name)
     end
 
+    def reply_to
+      facility.email
```

Comment:
Agreed, you could do something like `product.try(:contact_email).presence || facility.email`

It's a good catch on `#product` because that may be an otherwise dead method.